frontend_lib/src/component/DropdownMenu/DropdownMenu.jsx

```diff
@@ -11,13 +11,13 @@ const DropdownMenu = props => {
         aria-haspopup='true'
         className={classnames(
           'btn dropdown-toggle',
+          'dropdownMenuButton',
           props.isButton ? 'primaryColorBorder' : 'transparentButton',
           props.buttonCustomClass
         )}
         data-cy={props.buttonDataCy}
         data-toggle='dropdown'
         disabled={props.buttonDisabled}
-        id='dropdownMenuButton'
```

Comment:
question just to be explicit: normally bootstrap dropdown menus have an id, did you test to see if all menus still work fine without it?

Reply:
Are you sure? I see that in [the bootstrap doc for dropdowns](https://getbootstrap.com/docs/4.0/components/dropdowns/), we use the same id as their example, but I don't see anything mentioning ids being needed . Should bootstrap rely on ids, I'd pretty much expect them not to work on Tracim because at all times, we have several dropdowns displayed on the page with this same id and Bootstrap would probably pick the wrong one.

But yes, dropdowns still work fine :-)

Reply:
I didn't say it was necessary, maybe I expressed it wrong, I just meant that they use it in all the examples in the doc and I wanted to make sure that you had tested it. :slightly_smiling_face:

Reply:
I understood you right and you were right to do so :-)

But I just wanted to be sure on my part because you said "normally bootstrap dropdown menus have an I", which might have been a way to say that they should have one.